spec/features/profile_spec.rb

```diff
@@ -95,6 +95,38 @@ def mock_writing_to_filesystem
   end
 end
 
+RSpec.feature "Visit page with special role", :type => :feature do
+  before :each do
+    @profile = FactoryGirl.create(:profile)
+    @user = FactoryGirl.create(:user, role: :pupil, profile: @profile)
+    @event = FactoryGirl.create(:event)
+    @application_letter = FactoryGirl.create(:application_letter_accepted, user: @user, event: @event)
+    login_as(@user, scope: :user)
+    visit profile_path(@user.profile)
```

Comment:
you might be able to skip the user too, if you just need the profile. Not really sure, but since profile belongs to a user, one should be auto-created if you create a profile (maybe that changed tho, you gonna have to try it out)